src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public static boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null || repeatPassword == null) {
+            throw new PasswordValidationException("Wrong passwords");
+        } else if (password.equals(repeatPassword) && password.length() >= 10) {
+            return true;
+        }
```

Comment:
can we make this checking in the one `if` block?

Reply:
in one block now. =)